advocacy_docs/tools/alteruser_utility/index.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: "Alteruser utility"
+description: "How to use the alteruser utility to alter a PostgreSQL role in your EDB Postgres Advanced Server databases"
+iconName: Tools
+deepToC: true
+---
+
+## Overview
```

Comment:
Why is it necessary to say "existing" throughout this topic? Altering a role suggests the role must already exist. I recommend removing all but the first instance.

Reply:
removed existing